astropy/io/fits/connect.py

```diff
@@ -324,30 +311,6 @@ def _encode_mixins(tbl):
                         for attr in ('description', 'meta'))
                     for col in tbl.itercols())
 
-    # If PyYAML is not available then check to see if there are any mixin cols
-    # that *require* YAML serialization.  FITS already has support for Time,
-    # Quantity, so if those are the only mixins the proceed without doing the
-    # YAML bit, for backward compatibility (i.e. not requiring YAML to write
-    # Time or Quantity).  In this case other mixin column meta (e.g.
-    # description or meta) will be silently dropped, consistent with astropy <=
-    # 2.0 behavior.
-    try:
-        import yaml  # noqa
-    except ImportError:
-        for col in tbl.itercols():
-            if (has_info_class(col, MixinInfo) and
```

Comment:
lint: Since this change, `has_info_class` is no longer used in this module, so the import can be removed.  Ditto for `MixinInfo`

Reply:
Done!